commands/cmd-shell/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use anyhow::{Context, Result};
+use async_trait::async_trait;
+use clap::{ArgMatches, Clap};
+use oro_command::OroCommand;
+use oro_command::OroCommandLayerConfig;
+use oro_config::OroConfig;
+use oro_error_code::OroErrCode as Code;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+#[derive(Debug, Clap)]
+pub struct ShellCmd {
+    #[clap(long, default_value = "node")]
+    node: String,
+
+    #[clap(long)]
+    data_dir: Option<PathBuf>,
+
+    #[clap(multiple = true)]
+    args: Vec<String>,
```

Comment:
I believe in order to work properly, this might need to be the last argument? I forget how the magic VarArgs thing works.